scss/components/_activity-card.scss

```diff
@@ -111,9 +110,6 @@
 .civicase__activity-subject {
   color: $gray-darker;
   font-weight: $font-weight-heading;
-  overflow: hidden;
```

Comment:
Why are these removed?

Reply:
These are not removed. These are moved under `_activity-card--short.scss` as we need these styles for short activity card variant only.